radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/system/api.yaml

```diff
@@ -130,10 +130,10 @@ components:
       type: object
       required:
         - network_status
-        - current_fork_hash
+        - current_fork_name
```

Comment:
Still don't think this should be in the health endpoint, but in a dedicated endpoint instead.